src/libraries/react-calendar/components/CalendarMain.tsx

```diff
@@ -1,69 +1,41 @@
-import React, { useContext } from "react"
+import { useContext } from "react"
 import { CalendarContext } from "../common/CalendarContext"
-import { makeStyles, Theme } from "@material-ui/core/styles"
-import clsx from "clsx"
-import withWidth from "@material-ui/core/withWidth"
-import getWeekDays from "../common/getWeekDays"
+import { styled } from '@mui/system'
 import getSelectedWeekIndex from "../common/getSelectedWeekIndex"
 import CalendarLayoutMonth from "./CalendarLayoutMonth"
 import CalendarLayoutDayWeek from "./CalendarLayoutDayWeek"
+import { getDatesToDisplay } from "../utils"
 
-const drawerWidth = 260
-const useStyles = makeStyles((theme: Theme) => ({
-    drawerHeader: {
-        display: "flex",
-        alignItems: "center",
-        ...theme.mixins.toolbar,
-        justifyContent: "flex-center",
-    },
-    content: {
-        flexGrow: 1,
-        // padding: theme.spacing(0),
-        transition: theme.transitions.create("margin", {
-            easing: theme.transitions.easing.sharp,
-            duration: theme.transitions.duration.leavingScreen,
-        }),
-        marginLeft: -drawerWidth,
-        height: "100%",
-        width: "100%",
-        minWidth: 1000,
-    },
-    contentShift: {
-        transition: theme.transitions.create("margin", {
-            easing: theme.transitions.easing.easeOut,
-            duration: theme.transitions.duration.enteringScreen,
-        }),
-        marginLeft: 0,
-    },
-}))
+const MainCalendarWrapper = styled('div')({
+    flexGrow: 1,
+    height: "100%",
+    width: "100%",
+    minWidth: 1000,
+    marginTop: 64
+})
 
-function CalendarMain(props: any) {
-    const classes = useStyles()
-    const { stateCalendar } = useContext(CalendarContext)
-    const { selectedDate, locale, layout } = stateCalendar
 
-    const { open, runAnimation } = props
-    // const theme = useTheme()
+interface CalendarMainProps {
+    runAnimation: boolean
+}
 
-    const weeks = getWeekDays(selectedDate, 7)
-    const selectedWeekIndex = getSelectedWeekIndex(selectedDate, weeks, 0)
+function CalendarMain({ runAnimation }: CalendarMainProps) {
+    const { stateCalendar } = useContext(CalendarContext)
+    const { selectedDate, layout } = stateCalendar
+
+    const weeks = getDatesToDisplay(selectedDate)
+    const selectedWeekIndex = getSelectedWeekIndex(selectedDate, weeks)
     const selectedWeek = weeks[selectedWeekIndex]
 
     return (
-        <div
-            className={clsx(classes.content, {
-                [classes.contentShift]: open,
-            })}
-        >
-            <div className={classes.drawerHeader} />
-
-            {layout === "month" && <CalendarLayoutMonth weeks={weeks} runAnimation={runAnimation} locale={locale} />}
+        <MainCalendarWrapper>
```

Comment:
In this case you totally droped `classes.contentShift` which should apply if we have `open` value, `MainCalendarWrapper` should accept parameter open which apply different styles for it.

Reply:
The reason I deleted this is because in reality in the project the side bar with mini map could never be closed, because open/close button was hidden behind the toolbar at the top.

Please tell me if I need to implement opening and closing of the mini calendar.

Reply:
We don't need it, so I will resolve this comment.